### DISCUSSION TITLE
账号显示因为NSFW被封禁

### DISCUSSION THREAD

**Discussion by Feliks151450** [BurnHair/docs#130] [技术支持]
用户820显示被封禁了，可以帮忙看看具体是因为什么内容封禁的吗

**Comment by BennyThink** [comment]
内容：Translate the following source text to Simplified Chinese Language, Output translation directly without any additional text.
Source Text: Track6.監禁生活402日目②【パイズリ・おっぱいにぶかっけ・精液飲み】.mp3
Translated Text:

**Reply by Feliks151450** [reply]
OK，终于知道用户到底干了什么了。。。谢谢

**Reply by shuangmuaigwu** [reply]
6

**Reply by Passerby1011** [reply]
悬着的心终于死了

**Comment by yydlh** [comment]
用户3553已被封禁,是什么原因啊

**Reply by BennyThink** [reply]
已解决
